server/routes/dpr/index.ts

```diff
@@ -24,12 +25,18 @@ export function dprRouter(): Router {
             apiUrl: config.apis.hmppsIncidentReportingApi.url,
             apiTimeout: config.apis.hmppsIncidentReportingApi.timeout.deadline,
             layoutTemplate: 'partials/dprLayout.njk',
-            tokenProvider: defaultTokenProvider,
+            tokenProvider: (req: Request, res: Response, next: NextFunction) => {
+              return res.locals.systemToken
```

Comment:
overridden the default token provider as we don't want the user token but the system token with the username added.

There is more work to be done on the API side to use this token correctly to look up the users roles - this is future work but for now it works because I've add the roles to the system token (which actually are user roles) - these will be removed in future once API is corrected